src/main/kotlin/com/doist/gradle/androidtranslationscheck/AndroidTranslationsCheckExtension.kt

```diff
@@ -0,0 +1,11 @@
+package com.doist.gradle.androidtranslationscheck
+
+import org.gradle.api.Project
+import org.gradle.api.provider.Property
+import org.gradle.kotlin.dsl.property
+
+@Suppress("UnstableApiUsage")
+open class AndroidTranslationsCheckExtension(project: Project) {
+    val argumentRegex: Property<String> =
```

Comment:
Nit. Why would we need `Property<String>` instead of `var String`?

Reply:
It's to use the lazy configuration Gradle pattern: https://docs.gradle.org/current/userguide/lazy_configuration.html#lazy_properties

I saw it in a few other Gradle plugins and it seems a common practice.